recipes/libcurl/all/conanfile.py

```diff
@@ -189,31 +185,34 @@ def requirements(self):
         if self.options.with_c_ares:
             self.requires("c-ares/1.18.1")
 
+    def build_requirements(self):
+        if self._is_using_cmake_build:
+            if self._is_win_x_android:
+                self.build_requires("ninja/1.11.0")
+        else:
+            self.build_requires("libtool/2.4.6")
+            self.build_requires("pkgconf/1.7.4")
+            if self._settings_build.os == "Windows" and not get_env("CONAN_BASH_PATH"):
+                self.build_requires("msys2/cci.latest")
+
```

Comment:
keep build_requirement after validate please, it's really called after, and it's easier to reason about the global logic when methods are properly sorted.

Reply:
Sorry I had a doubt there and forgot to consult [the reviewing documentation](https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#order-of-methods-and-attributes).

Do you think the hook could check the method order to make sure all new PR respect it without bothering reviewers ?